docs/sdk.rst

```diff
@@ -370,23 +385,24 @@ are usually safe, though not universally.
 Errors may surface as serialization/deserialization ``Exception``s, Globus
 Compute task workers lost due to ``SEGFAULT``, or even incorrect results.
 
-Note that the |Client|_ class's :func:`~globus_compute_sdk.Client.register_function` method can be used
-to pre-serialize a function using the registering SDK's environment and
-return a UUID identifier.   The resulting bytecode will then be deserialized
-at run time by an Endpoint whenever a task that specifies this function UUID
-is submitted (possibly from a different SDK environment) using the Client's
-:func:`~globus_compute_sdk.Client.run` or the Executor's :func:`~globus_compute_sdk.Executor.submit_to_registered_function` methods.
-On the other hand, the |Executor|_ 's :func:`~globus_compute_sdk.Executor.submit` takes a function argument
-and serializes a fresh copy each time it is invoked.
+Note that the |Client| class's :func:`~globus_compute_sdk.Client.register_function`
+method can be used to pre-serialize a function using the registering SDK's environment
+and return a UUID identifier.  The resulting bytecode will then be deserialized at run
+time by an Endpoint whenever a task that specifies this function UUID is submitted
+(possibly from a different SDK environment) using the Client's
+:func:`~globus_compute_sdk.Client.run` or the |Executor|'s
+:py:meth:`~globus_compute_sdk.Executor.submit_to_registered_function` methods.
```

Comment:
is there a semantic difference between `:func:` and `:py:meth:`? if not, we should standardize to one. (I don't have a strong preference which, but I might suggest `:func:` simply because it's in use in other docs files, while I don't believe `:py:meth:` is in use elsewhere)

Reply:
If that's the distinction, we should use `:py:meth:` for every method and `:func:` _just_ for freestanding functions, even if just in the files that this PR touches for now

Reply:
Okay; not worth it for this PR.  Moved to `:func:`, and can discuss and/or do (not) at a later date.